design-documents/graph-ql/coverage/add-items-to-cart/AddSimpleProductToCart.graphqls

```diff
@@ -0,0 +1,78 @@
+type Mutation {
+    addSimpleProductsToCart(input: AddSimpleProductsToCartInput): AddSimpleProductsToCartOutput
+    updateSimpleProductsInCart(input: UpdateSimpleProductsInCartInput): UpdateSimpleProductsInCartOutput
+}
+
+input UpdateSimpleProductsInCartInput {
+    cart_id: String!
+    cartItems: [UpdateSimpleProductCartItemInput!]!
+}
+
+input UpdateSimpleProductCartItemInput {
+    details: UpdateCartItemDetailsInput!
+    customizable_options:[CustomizableOptionInput]
+}
+
+input AddSimpleProductsToCartInput {
+    cart_id: String!
+    cartItems: [SimpleProductCartItemInput!]!
+}
+
+input SimpleProductCartItemInput {
+    details: CartItemDetailsInput!
+    customizable_options:[CustomizableOptionInput!]
+}
+
+input CartItemDetailsInput {
+    sku: String!
+    quantity: Float!
+}
+
+input UpdateCartItemDetailsInput {
+    cart_item_id: String!
+    quantity: Float
+}
+
+input CustomizableOptionInput {
+    id: Int!
+    value: String!
+}
+
+type AddSimpleProductsToCartOutput {
+    cart: Cart!
+}
+
+type Cart {
+    items: [CartItemInterface]
+}
+
+interface CartItemInterface @typeResolver(class: "Magento\\CatalogCheckoutGraphQl\\Model\\CartItemInterfaceTypeResolverComposite") {
+    id: String! # Hashed cart item ID
+    qty: Float!
+    product: ProductInterface!
+}
+
+type SimpleCartItem implements CartItemInterface {
+    customizable_options: [SelectedCustomizableOption]
+}
+
+type SelectedCustomizableOption {
+    id: Int!
+    label: String!
+    type: String!
+    values: [SelectedCustomizableOptionValue!]!
+    sort_order: Int!
+}
+
+type SelectedCustomizableOptionValue {
+    id: Int
+    label: String!
+    price: CartItemSelectedOptionValuePrice!
```

Comment:
Do we need price info?

Maybe id and value will be enough for the response about selected options
```
{
   "id": 14,
   "value": "test",                    
 }
```